man/man5/zfs-module-parameters.5

```diff
@@ -1723,7 +1761,8 @@ Default value: \fB75\fR.
 \fBzvol_inhibit_dev\fR (uint)
 .ad
 .RS 12n
-Do not create zvol device nodes
+Do not create zvol device nodes. This can dramatically speed up import times
+for pools with many zvols.
```

Comment:
These days this is less likely to have an impact on import times since the device nodes are created asynchronously from the `zpool import`.